src/components/Layout/AppLayout.tsx

```diff
@@ -64,18 +60,19 @@ export const AppLayout = ({
       areas={mobileAreas || defaultMobileAreas}
       areasMd={desktopAreas || defaultDesktopAreas}
       areasLg={desktopAreas || defaultBigscreenAreas}
-      templateRows={"60px 1fr 30px"}
+      height={height}
+      templateRows={"auto 1fr auto"}
```

Comment:
nit: using `{"ish"}` is doing two calculations, strongly recommend updating all `{"` to just `"`
```suggestion
      templateRows="auto 1fr auto"
```

Reply:
Ah, good catch! done.